boards/arm/nucleo_f446re/doc/nucleof446re.rst

```diff
@@ -0,0 +1,216 @@
+- 12-bit ADC(3) with 16 channels
+- 12-bit DAC with 2 channels
+
+
+
+More information about STM32F446RE can be found here:
+       - `STM32F446RE on www.st.com`_
+       - `STM32F446 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr nucleo_f446re board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/nucleo_f446re/nucleo_f446re_defconfig``
+
+
+Connections and IOs
+===================
+
+Nucleo F446RE Board has 8 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+Available pins:
+---------------
+.. image:: img/nucleo_f446re_arduino_top_left.png
+     :width: 720px
+     :align: center
+     :height: 540px
+     :alt: Nucleo F446RE Arduino connectors (top left)
+.. image:: img/nucleo_f446re_arduino_top_right.png
+     :width: 720px
+     :align: center
+     :height: 540px
+     :alt: Nucleo F446RE Arduino connectors (top right)
+.. image:: img/nucleo_f446re_morpho_top_left.png
+     :width: 720px
+     :align: center
+     :height: 540px
+     :alt: Nucleo F446RE Morpho connectors (top left)
+.. image:: img/nucleo_f446re_morpho_top_right.png
+     :width: 720px
+     :align: center
+     :height: 540px
+     :alt: Nucleo F446RE Morpho connectors (top right)
+
+For mode details please refer to `STM32 Nucleo-64 board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+- UART_1_TX : PB6
+- UART_1_RX : PB7
+- UART_2_TX : PA2
+- UART_2_RX : PA3
+- USER_PB : PC13
+- LD2 : PA5
+- I2C1_SDA : PB9
+- I2C1_SCL : PB8
+- I2C2_SDA : PB3
+- I2C2_SCL : PB10
+- I2C3_SDA : PB4
+- I2C3_SCL : PA8
+
+System Clock
+------------
+
+Nucleo F446RE System Clock could be driven by internal or external oscillator,
+as well as main PLL clock. By default System clock is driven by PLL clock at 84MHz,
```

Comment:
Some more missing articles:

    as the main

    By default, the System clock is driven by the PLL clock at 84MHz, driven by an
    8MHz high-speed external clock.